src/discordtipbot/statistics.cr

```diff
@@ -0,0 +1,43 @@
+struct Statistics
+  DB.mapping({
+    transactions: {
+      type: Int64,
+      key:  "transaction_count",
+    },
+    total: {
+      type: BigDecimal,
+      key:  "transaction_sum",
+    },
+    tips: {
+      type: BigDecimal,
+      key:  "tip_sum",
+    },
+    soaks: {
+      type: BigDecimal,
+      key:  "soak_sum",
+    },
+    rains: {
+      type: BigDecimal,
+      key:  "rain_sum",
+    },
+  })
+
+  @@ttl : Time::Span = 1.minutes
+
+  @@last = Time.utc_now
```

Comment:
Not sure why you bother with `utc`.

Reply:
Just gives it a nicer look IMO 😃

Reply:
Compared to what? Just `Time.now`? 😕

Reply:
Yes. (Especially considering it's globally used, having it the same no matter where I move the VPS etc)

Reply:
But it literally doesn't matter since this value is just used internally within the class, compared relatively with ttl.